modules/prefixer.js

```diff
@@ -385,13 +389,19 @@ var getPrefixedStyle = function (style, mode /* 'css' or 'js' */) {
       // Ignore unsupported properties
       /* eslint-disable no-console */
       if (console && console.warn) {
-        console.warn('Unsupported CSS property "' + property + '"');
+        var componentContext = component
+          ? '" in component "' + component.constructor.displayName + '"'
+          : '';
+
+        console.warn(
+          'Unsupported CSS property "' + property + componentContext
```

Comment:
Missing quotes if no component. Remove it from above and do this here (using template strings for clarity):

`Unsupported CSS property "${property}"` + componentContext